conformance/system_macros/add_macros.ion

```diff
@@ -0,0 +1,9 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// add_macros can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
```

Comment:
Consider coining a phrase that means "name, address, qualified address, qualified name, binary address, and binary system e-expression op code" and adding it to the glossary. Then you can avoid repeating this clause in each file.

Reply:
Like "macro reference", perhaps?

Reply:
Sure!